modules/scanning-images.adoc

```diff
@@ -58,80 +58,96 @@ IMPORTANT: If you are using a Red Hat link:https://catalog.redhat.com/software/c
 
 |===
 
-[discrete]
+[id="supported-package-formats_{context}"]
 == Supported package formats
 
 Scanner can check for vulnerabilities in images that use the following package formats:
 
-* yum
-* microdnf
 * apt
 * apk
 * dpkg
-* RPM
+* rpm
 
-[discrete]
+[id="supported-programming-languages_{context}"]
 == Supported programming languages
 
 Scanner can check for vulnerabilities in dependencies for the following programming languages:
 
+* Go (Scanner V4 only)
+** Binaries: The standard library version used to build the binary is analyzed. If the binaries are built with module support (go.mod), then the dependencies are also analyzed.
 * Java
-* JavaScript
+** JAR/WAR/EAR
+* JavaScript/Node.js
+** npm package.json
 * Python
+** egg and wheel formats
 * Ruby
+** gem
 
-[discrete]
+[id="supported-runtimes-frameworks_{context}"]
 == Supported runtimes and frameworks
 
-Beginning from {product-title} 3.0.50 (Scanner version 2.5.0), Scanner identifies vulnerabilities in the following developer platforms:
+Beginning from {product-title} 3.0.50 (Scanner version 2.5.0), the StackRox Scanner identifies vulnerabilities in the following developer platforms:
 
 * .NET Core
 * ASP.NET Core
 
-[discrete]
+These are not supported by Scanner V4.
+
+[id="supported-operating-systems_{context}"]
 == Supported operating systems
 
 The supported platforms listed in this section are the distributions in which Scanner identifies vulnerabilities, and it is different from the supported platforms on which you can install {product-title}.
 
 Scanner identifies vulnerabilities in images that contain the following Linux distributions:
 
 |===
-| Distribution | Version
+| Distribution | Version | Vulnerability database
 
 | link:https://www.alpinelinux.org/[Alpine Linux]
-| `alpine:v3.2`, `alpine:v3.3`, `alpine:v3.4`, `alpine:v3.5`, `alpine:v3.6`, `alpine:v3.7`, `alpine:v3.8`, `alpine:v3.9`, `alpine:v3.10`, `alpine:v3.11`, `alpine:v3.12`, `alpine:v3.13`, `alpine:v3.14`, `alpine:v3.15`, `alpine:v3.16`, `alpine:v3.17`, `alpine:v3.18`, `alpine:edge`
+| `alpine:3.2`[1],`alpine:3.3`, `alpine:3.4`, `alpine:3.5`, `alpine:3.6`, `alpine:3.7`, `alpine:3.8`, `alpine:3.9`, `alpine:3.10`, `alpine:3.11`, `alpine:3.12`, `alpine:3.13`, `alpine:3.14`, `alpine:3.15`, `alpine:3.16`, `alpine:3.17`, `alpine:3.18`, `alpine:3.19` (supported in Scanner V4), `alpine:edge`
+| National Vulnerability Database (NVD)
 
 | link:https://aws.amazon.com/amazon-linux-ami[Amazon Linux]
-| `amzn:2018.03`, `amzn:2`
+| `amzn:2018.03`, `amzn:2`, `amzn:2023`[2]
+| NVD
 
-| link:https://wiki.centos.org/Manuals/ReleaseNotes[CentOS]
-| `centos:6`, `centos:7`, `centos:8`
+| CentOS
+| `centos:6`[1], `centos:7`[1], `centos:8`[1]
+| No official vulnerability feed exists.
 
 | link:https://www.debian.org/releases/[Debian]
 | `debian:10`, `debian:11`, `debian:12`, `debian:unstable`
```

Comment:
can you add [`"Distroless"`](https://github.com/GoogleContainerTools/distroless)? It's supported in both scanners